internal/update/realtime/realtime.go

```diff
@@ -172,7 +172,13 @@ type updateStopTimesInDBArgs struct {
 // updateStopTimesInDB updates the trip stop times in the database. The boolean return value indicates
 // whether the path of the trip has changed.
 func updateStopTimesInDB(ctx context.Context, updateCtx common.UpdateContext, args updateStopTimesInDBArgs) (bool, error) {
-	populateStopSequences(args.trip, args.existingStopTimes, args.stopIDToPk)
+	// Check to see if stop sequences should be reassigned by transiter (even if present in GTFS).
+	var reassignStopSequences = false
+	if updateCtx.FeedConfig != nil && updateCtx.FeedConfig.GtfsRealtimeOptions != nil {
+		reassignStopSequences = updateCtx.FeedConfig.GtfsRealtimeOptions.ReassignStopSequences
+	}
```

Comment:
I wonder can we can replace this block with:

reassignStopSequences := updateCtx.FeedConfig.GetGtfsRealtimeOptions().GetReassignStopSequences()

This would require FeedConfig to always be non-nil though, which maybe it isn't in tests. Feel free to ignore this suggestion :)

Reply:
I think should be OK - I think my initial concern was if there was ever a case where a `FeedConfig` wouldn't be given in production (tests should always provide one), but I think this isn't possible.